docs/timeline/index.html

```diff
@@ -461,6 +461,18 @@ <h3 id="groups">Groups</h3>
       <td>no</td>
       <td>Provides a means to toggle the whether a group is displayed or not. Defaults to <code>true</code>.</td>
     </tr>
+    <tr>
+      <td>nestedGroups</td>
```

Comment:
subGroups refer to groups in a group and not nested underneath. So I actually prefer nestedGroups.